flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -447,6 +447,43 @@ var A320_Neo_UpperECAM;
                             }
                         ]
                     },
+                    {
+                        name: "ENG",
+                        messages: [
+                            {
+                                id: "thr_levers_not_set_flex",
+                                message: "THR LEVERS NOT SET",
+                                level: 2,
+                                alwaysShowCategory: true,
+                                isActive: () => (
+                                    this.getCachedSimVar("L:A32NX_AUTOTHRUST_THRUST_LEVER_WARNING_FLEX", "Bool")
+                                ),
+                                actions: [
+                                    {
+                                        style: "action",
+                                        message: "THR LEVERS",
+                                        action: "MCT/FLX",
+                                    }
+                                ]
+                            },
+                            {
+                                id: "thr_levers_not_set_toga",
+                                message: "THR LEVERS NOT SET",
+                                level: 2,
+                                alwaysShowCategory: true,
+                                isActive: () => (
+                                    this.getCachedSimVar("L:A32NX_AUTOTHRUST_THRUST_LEVER_WARNING_TOGA", "Bool")
+                                ),
+                                actions: [
+                                    {
+                                        style: "action",
+                                        message: "THR LEVERS",
+                                        action: "TOGA",
+                                    }
+                                ]
```

Comment:
These should be merged into a single warning. You can use actions.isCompleted to hide the non-applicable ones.